js/pages/cohort-definitions/cohort-definition-manager.js

```diff
@@ -110,7 +110,7 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
 
 			this.cohortDefinitionCaption = ko.computed(() => {
 				if (this.model.currentCohortDefinition()) {
-					if (this.model.currentCohortDefinition().id() === 0) {
+					if (this.model.currentCohortDefinition().id() == 0) {
```

Comment:
Describe a test case for cohort definition changes pls: cannot observe issue with current codebase